keyboards/akb/ogrn/readme.md

```diff
@@ -0,0 +1,27 @@
+# akb/ogrn
+
+![akb/ogrn](https://i.imgur.com/qbolyMa.png)
```

Comment:
This uses a lower resolution (but still pretty high) image that is more suitable with markdown rendering.
```suggestion
![akb/ogrn](https://i.imgur.com/qbolyMal.png)
```